src/jsp/com/cubrid/jsp/TargetMethod.java

```diff
@@ -201,14 +204,26 @@ private static void initdescriptorMap() {
         descriptorMap.put("double", "D");
     }
 
+    private static final String[] DUMMY_STRING_ARRAY = new String[0];
+
     public Method getMethod()
             throws SecurityException, NoSuchMethodException, ClassNotFoundException {
         Class<?> c = getClass(className);
         if (c == null) {
-            throw new ClassNotFoundException (className);
+            throw new ClassNotFoundException(className);
+        }
+        try {
+            return c.getMethod(methodName, argsTypes);
+        } catch (NoSuchMethodException e) {
+            List<String> argTypeNameList =
+                    Arrays.stream(argsTypes)
+                            .map(cl -> cl.getTypeName())
+                            .collect(Collectors.toList());
```

Comment:
trivial: please note that target JDK version of CUBRID 10.2 and 11.0 is 6. Arrays.stream() doesn't support on JDK 6. some additional works needs.

Reply:
OK. Thank you.